package.json

```diff
@@ -35,7 +35,7 @@
     "redux-mock-store": "^1.5.4",
     "resolve": "1.20.0",
     "resolve-url-loader": "3.1.2",
-    "sass-loader": "10.1.1",
+    "sass-loader": "10.2.0",
```

Comment:
This small change, but I see 16,316 additions, 53,247 deletions  in the package-lock file. Something feels wrong.

Reply:
This was leftover from me having other node issues and trying to resolve errors.